aten/src/ATen/native/Loss.cpp

```diff
@@ -314,53 +315,101 @@ Tensor& binary_cross_entropy_backward_out_cpu(const Tensor& grad, const Tensor&
     return grad_input;
 }
 
-Tensor binary_cross_entropy_with_logits(const Tensor& input, const Tensor& target, const c10::optional<Tensor>& weight_opt, const c10::optional<Tensor>& pos_weight_opt, int64_t reduction) {
+Tensor binary_cross_entropy_with_logits(
```

Comment:
Why does this function needs to change?
From the comment above, I though that only the backward needed changes?